coda/coda_mdstore/views.py

```diff
@@ -670,21 +670,44 @@ def externalIdentifierSearch(request, identifier=None):
 def externalIdentifierSearchJSON(request):
     """External_Identifier search formatted in JSON."""
     ark = request.GET.get('ark', '')
+    showAll = request.GET.get('showAll', False)
```

Comment:
Perhaps the default should be a `str` such as `'false'`, since if there is a value, it would be a `str`.